.github/workflows/codeql-build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
```

Comment:
@ArielSAdamsNASA can you remove the matrix build strategy and just set the environment variable using the `env` property? You can follow the format used to set the `SIMULATION=native` example. 

You can amend and force push the commit.

Reply:
Updated.